StyleCop.Analyzers/StyleCop.Analyzers/LayoutRules/SA1518CodeMustNotContainBlankLinesAtEndOfFile.cs

```diff
@@ -28,20 +28,30 @@ internal class SA1518CodeMustNotContainBlankLinesAtEndOfFile : DiagnosticAnalyze
         /// The ID for diagnostics produced by the <see cref="SA1518CodeMustNotContainBlankLinesAtEndOfFile"/> analyzer.
         /// </summary>
         public const string DiagnosticId = "SA1518";
-        private const string Title = "Code must not contain blank lines at end of file";
-        private const string MessageFormat = "Code must not contain blank lines at end of file";
-        private const string Description = "The code file has blank lines at the end.";
+        private const string TitleForAllowSetting = "Code must not contain blank lines at end of file";
```

Comment:
:exclamation: The title appears in the rule set editor, so it needs to be a fixed value. Only the message and description can change.